### DISCUSSION TITLE
`stable` branch

### DISCUSSION THREAD

**Discussion by vurvdev** [wiremod/wire#2936] [Suggestions]
If people want to keep complaining about wiremod being broken, I suppose we can return to the old model of wiremod not being updated for months since we have canary now for the people who care.

I propose a new branch of wiremod being created named `stable` or something like that, that occasionally pulls in `master` and auto-updates with a github action. Can cherry pick fixes this way too

Mostly @Denneisk was proposing this for a long time but I've kinda been brushing it off

**Comment by Denneisk** [comment]
> not being updated for months

I was expecting a branch to mirror the Workshop uploads, which currently is roughly monthly. Maybe I'm just mis-interpreting.

My problem is that I'm worried not enough people would use the Canary release and report bugs. Maybe, for those who want it, have a Wiremod ESR that's updated even slower? Lol.

**Reply by vurvdev** [reply]
> I was expecting a branch to mirror the Workshop uploads, which currently is roughly monthly. Maybe I'm just mis-interpreting.

The "stable" workshop would mirror this branch

> My problem is that I'm worried not enough people would use the Canary release and report bugs. Maybe, for those who want it, have a Wiremod ESR that's updated even slower? Lol.

True, but we're mostly relying on servers using the github version to report bugs right now, tbf

**Reply by Denneisk** [reply]
> The "stable" workshop would mirror this branch

In that case, I have no issues.

> we're mostly relying on servers using the github version to report bugs right now, tbf

Also true. Should consider propaganda strategies to encourage bug reporting.
